cumulus/pallets/xcmp-queue/src/lib.rs

```diff
@@ -791,22 +788,41 @@ impl<T: Config> XcmpMessageHandler for Pallet<T> {
 							},
 						}
 					},
-				XcmpMessageFormat::ConcatenatedVersionedXcm =>
+				XcmpMessageFormat::ConcatenatedVersionedXcm => {
+					let mut new_page = true;
```

Comment:
Could please add here some docs or explain, what "new_page" means in this context?
Plus also add some docs why we need `new_page` reasoning in `fn enqueue_xcmp_message(size_in_bytes: usize, new_page: bool)`.

I am not sure if I understand it correctly, so basically this PR is about overestimating processing the first (or second)?

Reply:
Done. PTAL ! Added a comment here and one in the `fn enqueue_xcmp_message(size_in_bytes: usize, new_page: bool)` benchmark